clusters/production/overlay/third-party/snyk-monitor/helmRelease.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   chart:
     spec:
-      version: 1.80.0 # https://artifacthub.io/packages/helm/snyk/snyk-monitor
+      version: 1.81.1 # https://artifacthub.io/packages/helm/snyk/snyk-monitor
```

Comment:
The version running in prod is `1.82.0-approved`, specified by the `.image` tag. The version should be set to `1.82.0` and the `.image` tag should be removed.